src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -829,8 +848,11 @@ private function camelize($string)
      *
      * @return array|null An array containing the adder and remover when found, null otherwise
      */
-    private function findAdderAndRemover(\ReflectionClass $reflClass, array $singulars)
+    private function findAdderAndRemover(\ReflectionClass $reflClass, array $singulars, PropertyMetadata $metadata = null)
     {
+        if ($metadata && $metadata->getAdder() && $metadata->getRemover()) {
```

Comment:
I'll try to fix one of the accessor and let the component guess the other one, seems reasonable